src/components/tableau/tableau.js

```diff
@@ -1,10 +1,40 @@
 import React, { Component } from 'react';
 
+var tableauScriptUrl = "http://public.tableau.com/javascripts/api/tableau-2.0.0.min.js";
```

Comment:
I can change this. but right here we would want it to be const instead of var because it something that should be unchangeable, but I like I'm going to put all the api urls in a config file anyway.